spec/game_spec.rb

```diff
@@ -0,0 +1,79 @@
+require_relative '../lib/game'
+
+describe Game do
+  let!(:game) { Game.new }
+  describe "#won?" do
+    context 'when X X X across the top row' do
+      it 'returns true' do
+        game.board.update_board('X',1)
+        game.board.update_board('X',2)
+        game.board.update_board('X',3)
+        expect( game.won?("X") ).to eql(true)
+      end
+
+      it 'returns false for O player' do
+        expect( game.won?("O") ).to eql(false)
```

Comment:
In all of these blocks where it says `returns false for O/X player`, will return false no matter what, because you are not giving your test a board to check for the winning condition, I would suggest to pass on the same parameters as the `it returns true`. So that your test is actually checking for a winning condition. I ran the test with different values and was returning green every time.